modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/model/impl/CommerceOrderImpl.java

```diff
@@ -44,6 +47,17 @@
  */
 public class CommerceOrderImpl extends CommerceOrderBaseImpl {
 
+	@Override
+	public AccountEntry getAccountEntry() throws PortalException {
```

Comment:
@brianikim FYI I added getAccountEntry() to CommerceOrder, might have some overlap with the headless modules you are working on.